gtsfm/view_graph_estimator/cycle_consistent_rotation_estimator.py

```diff
@@ -0,0 +1,237 @@
+"""A ViewGraphEstimator implementation which ensures relative rotations are consistent in the cycles of the graph.
+
```

Comment:
@akshay-krishnan can we add back in these original references?

```
    Based off of:
        https://github.com/sweeneychris/TheiaSfM/blob/master/src/theia/sfm/filter_view_graph_cycles_by_rotation.cc
    See also:
        C. Zach, M. Klopschitz, and M. Pollefeys. Disambiguating visual relations using loop constraints. In CVPR, 2010
        http://people.inf.ethz.ch/pomarc/pubs/ZachCVPR10.pdf
        Enqvist, Olof; Kahl, Fredrik; Olsson, Carl. Non-Sequential Structure from Motion. ICCVW, 2011.
        https://portal.research.lu.se/ws/files/6239297/2255278.pdf
```

Reply:
Added to the class documentation below.

Reply:
thanks!